### DISCUSSION TITLE
I am seeing an error

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1093] [General]
I was working on my site tonight and when I updated my homepage this appeared on my sidebar:

Notice: Undefined index: types in /home/izgr1zclaxch/public_html/wp-content/plugins/12-step-meeting-list/includes/shortcodes.php on line 30
Notice: Undefined index: types in /home/izgr1zclaxch/public_html/wp-content/plugins/12-step-meeting-list/includes/shortcodes.php on line 36

This is my site https://alanonalateen6nc.org/

How do I correct this?

Rose L
District 6

**Comment by joshreisner** [comment]
hi Rose, sounds like we have a bug that is showing that notice when one of the upcoming meetings has no types and debug mode is on. it's harmless, but i'll try to confirm it on our end.

i'm not seeing it anymore, so my guess is the affected meeting probably just passed. it might be this one: https://alanonalateen6nc.org/meetings/new-al-anon-meeting/ - so a non-technical but kludgy way to fix it would be to see if you can apply at least one type to every meeting for now, like Open / Closed.

a better way to solve this but which is slightly more technical is to see if "debug" mode is on for some reason. if you want to use the file manager to check your `wp-config.php` file you may find that `WP_DEBUG` is set to `true` - you want that to be set to `false` in production

here is one hosting company's instructions on how to do this: https://www.domain.com/help/article/wordpress-how-to-enable-or-disable-wp-debug -- around step 5 the instructions are worth reading

**Reply by joshreisner** [reply]
yep, i was able to reproduce this on our demo site. has to be an in-person meeting

<img width="387" alt="Screenshot 2023-03-16 at 5 26 29 PM" src="https://user-images.githubusercontent.com/1551689/225781336-aec66ea3-44a2-48d2-a89f-45e21fca3612.png">
